src/ray/raylet/node_manager.h

```diff
@@ -25,7 +26,7 @@ struct NodeManagerConfig {
   ResourceSet resource_config;
   int num_initial_workers;
   int num_workers_per_process;
-  std::vector<std::string> worker_command;
+  std::unordered_map<Language, std::vector<std::string>> worker_command;
```

Comment:
Since enum can be treated as int, is it possible to use vector with enum as index?

Reply:
That's doable. But I prefer using a map. Because:
1) code is more clear: `worker_command[Language::PYTHON]` vs `worker_command[static_cast<int>(Language::PYTHON]`.
1) static type checking: you cannot pass invalid values.

also, performance wise, I think there's no difference?